### PR TITLE
Update Hebrew translations

### PR DESCRIPTION
**Pull request by omeritzics**

### CODE REVIEW

File: locales/he_IL/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -99,14 +101,16 @@ msgid ""
 "%sClick here%s to try again, if you think there should be results for this "
 "search."
 msgstr ""
+"%sלחץ כאן%s כדי לנסות שוב, if you think there should be results for this "
+"search."
```
Comment (GioSensation) [inline locales/he_IL/LC_MESSAGES/duckduckgo.po:105]: I think you forgot to complete this translation. Would you mind taking another look?

File: locales/he_IL/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -2392,7 +2396,7 @@ msgstr "מידע נוסף"
 
 msgctxt "Ads GDPR, internal use only. Do not change"
 msgid "More Info"
-msgstr ""
+msgstr "מידע נוסף"
```
Comment (GioSensation) [inline locales/he_IL/LC_MESSAGES/duckduckgo.po:2399]: This string is for internal use only and can't be changed, thanks.
```suggestion
msgstr ""
```

**Comment by GioSensation** [comment]
Thanks for your contribution @omeritzics . Much appreciated. Do you mind checking my comment on that one string? Everything else looks good.

**Comment by omeritzics** [comment]
Yes, I'll fix that string.

**Comment by omeritzics** [comment]
I'll open a new PR with the fixed string.

**Pull request closed without merging**